### PR TITLE
DM-31364: Add Patch Thursday messages for IDF

### PR DESCRIPTION
**Pull request by jonathansick**
The `pre_patch_thursday` and `patch_thursday` messages have a recurring schedule, so they should trigger automatically.

The patch_thursday_extended message can be manually enabled to show a special patch thursday message after the window has elapsed to tell folks that we're still doing work.

### CODE REVIEW

**Comment by frossie** [comment]
It's a bit too wordy I think but let's go for today and we can iterate later.

**Review by frossie — APPROVED** [review]

**Comment by jonathansick** [comment]
@frossie Thanks. Is the issue the body text? That was an extract from the community post, but I'm happy to slim it down.
